### ISSUE TITLE
Source control issue while merge

### ISSUE THREAD

**Issue by HossamHoussien** [Microsoft/vscode#58752]
Issue Type: <b>Bug</b>

After using the source control feature to pull files from a remote repo, there was a conflict in some files but I didn't get the usual message that indicates that a conflict has occurred 
Ex: "
<<<head ....
<<< Incoming changes ... 

It was working great before last vs code update, kindly fix it.

VS Code version: Code 1.27.2 (f46c4c469d6e6d8c46f268d1553c5dc4b475840f, 2018-09-12T16:17:45.060Z)
OS version: Windows_NT x64 10.0.17134

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Athlon(tm) Dual Core Processor 5000B (2 x 2593)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: disabled_off<br>native_gpu_memory_buffers: disabled_software<br>rasterization: unavailable_software<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|5.73GB (1.90GB free)|
|Process Argv|D:\Installed Programs\Microsoft VS Code\Code.exe|
|Screen Reader|no|
|VM|67%|

</details><details><summary>Extensions (17)</summary>

Extension|Author (truncated)|Version
---|---|---
laravel-goto-view|cod|1.2.1
bracket-pair-colorizer|Coe|1.0.60
vscode-eslint|dba|1.6.0
vscode-html-css|ecm|0.2.0
php-debug|fel|1.12.5
php-intellisense|fel|2.3.5
auto-close-tag|for|0.5.6
beautify|Hoo|1.4.3
mssql|ms-|1.4.0
vetur|oct|0.12.6
laravel-blade|one|1.18.0
laravel5-snippets|one|1.5.0
material-icon-theme|PKi|3.6.0
laravel-helpers|raf|0.2.2
php-formatter|Sop|0.2.4
bootstrap4-vscode|the|4.3.0
html-css-class-completion|Zig|1.17.1

</details>

**Comment by joaomoreno** [comment]
Can you reproduce the issue? Note that the `<<<<` markers only appears in **both modified** conflicts. There are many other conflict types.

**Comment by HossamHoussien** [comment]
Yes, I'm talking about "Both modified" conflict. Usually, in previous releases, vscode was notifying you that there is a "both modified" conflict has occurred and list all files that contain that kind of conflict but in the last release I faced a problem where the list of conflict files doesn't appear and I had to figure which files contain these conflicts manually.

**Comment by joaomoreno** [comment]
> vscode was notifying you that there is a "both modified" conflict has occurred

Can you show me what this notification looks like? You can use last month's release to see it: https://az764295.vo.msecnd.net/stable/493869ee8e8a846b0855873886fc79d480d342de/VSCode-win32-x64-1.26.1.zip
